lib/tasks/end_to_end.ex

```diff
@@ -14,7 +15,21 @@ defmodule Mix.Tasks.EndToEnd do
     )
   end
 
+  @npm_chromedriver_package "chromedriver"
+  defp chrome_outdated?() do
+    with {raw_outdated_modules, _status_code} <- System.cmd("npm", ["outdated", "-g", "--json"]),
+         {:ok, parsed_outdated_modules} <- Jason.decode(raw_outdated_modules) do
+      if Map.has_key?(parsed_outdated_modules, @npm_chromedriver_package) do
+        print_red_line "CHROME IS OUTDATED: Tests will likely fail. Please run `npm outdated -g --json for more information."
```

Comment:
Steven! First of all, thanks for taking this on in a thoughtful way. It's needed and appreciated.

Regarding the language here, is it that Chrome is updated, or chromedriver?

Reply:
Do we want to be more explicit and instruct the developer to run `npm install chromedriver -g` if we detect that it's outdated?

Reply:
> Tests will likely fail

I think we might want to make the language here a little less confident, as I was able to run the tests, _get the warning output_ due to being 1 patch behind chromedriver stable, and then the tests ran and passed. 

Maybe `Tests may fail due to not having the latest chromedriver and Chrome installed on your system. Please update accordingly.`? Or something to that effect. Thoughts?

Reply:
From a local run:

```
$ mix e2e
warning: CHROME IS OUTDATED: Tests will likely fail. Please run `npm outdated -g --json for more information.
  lib/tasks/end_to_end.ex:6: Mix.Tasks.EndToEnd.run/1
  (mix) lib/mix/task.ex:331: Mix.Task.run_task/3
  (mix) lib/mix/task.ex:365: Mix.Task.run_alias/3
  (mix) lib/mix/task.ex:292: Mix.Task.run/2
  (mix) lib/mix/cli.ex:79: Mix.CLI.run_task/2
  (elixir) src/elixir_compiler.erl:79: :elixir_compiler.dispatch/4
  (elixir) src/elixir_compiler.erl:63: :elixir_compiler.compile/3
  (elixir) src/elixir_lexical.erl:17: :elixir_lexical.run/2
  (elixir) src/elixir_compiler.erl:23: :elixir_compiler.quoted/2
  (elixir) lib/code.ex:767: Code.require_file/2
  (elixir) lib/kernel/cli.ex:503: Kernel.CLI.wrapper/1
  (elixir) lib/enum.ex:1327: Enum."-map/2-lists^map/1-0-"/2
  (elixir) lib/kernel/cli.ex:68: Kernel.CLI.process_commands/1
  (elixir) lib/kernel/cli.ex:26: anonymous fn/2 in Kernel.CLI.main/1
  (elixir) lib/kernel/cli.ex:105: anonymous fn/3 in Kernel.CLI.exec_fun/2


> @ compile-test /Users/vanderhoop/dev/remote_retro
> webpack --config='webpack.dll.js' && webpack --config='webpack.shared.config.js'

[hardsource:73288e8a] Using 5 MB of disk space.
[hardsource:73288e8a] Tracking node dependencies with: yarn.lock.
[hardsource:73288e8a] Reading from cache 73288e8a...
Hash: 7f4d2f5320f5ea5c7ea1
Version: webpack 4.29.6
Time: 1595ms
Built at: 02/20/2020 5:56:15 AM
                                Asset       Size  Chunks             Chunk Names
                          css/app.css   38.5 KiB    main  [emitted]  main
                          favicon.ico  979 bytes          [emitted]
               images/action-item.svg   2.32 KiB          [emitted]
          images/apple-touch-icon.png   10.6 KiB          [emitted]
                  images/confused.svg   3.41 KiB          [emitted]
                  images/continue.svg   1.54 KiB          [emitted]
                     images/happy.svg  539 bytes          [emitted]
                       images/sad.svg   1.57 KiB          [emitted]
                     images/start.svg   1.54 KiB          [emitted]
                      images/stop.svg   1.56 KiB          [emitted]
               images/stride_logo.png     14 KiB          [emitted]
             images/traffic_light.svg   2.13 KiB          [emitted]
         images/trusted_by/casper.svg   5.06 KiB          [emitted]
images/trusted_by/flatiron_school.png   5.51 KiB          [emitted]
           images/trusted_by/sony.svg    2.7 KiB          [emitted]
    images/trusted_by/squarespace.svg    5.4 KiB          [emitted]
   images/trusted_by/warby_parker.svg   5.13 KiB          [emitted]
images/trusted_by/weather_channel.svg   9.05 KiB          [emitted]
                            js/app.js    556 KiB    main  [emitted]  main
                           robots.txt  207 bytes          [emitted]

Executing end-to-end tests...

	Note: browser logs will be written to `browser_logs.log`

Excluding tags: [:test, :skip]
Including tags: [:feature_test]

......................

Finished in 33.3 seconds
22 tests, 0 failures

Randomized with seed 938133
```

Reply:
I have taken your suggested copy but kept a pithy error of `chromedriver is out of date` at the beginning of the line.